md/runtime-monitoring.md

```diff
@@ -0,0 +1,108 @@
+# Bonita Runtime Monitoring
+
+Discover how to monitor a runtime environment running Bonita 
+
+## Why monitoring ?
+
+Monitoring a Production environment is crucial to ensure the runtime is correctly sized and tuned.
+
+Bonita provides a series of technical and BPM-related metrics to monitor the health of Bonita runtime environment.  
+Some metrics are enabled by default and cannot be disabled. Some others are optional and can be enabled according to
+your needs.
+
+## Glossary
```

Comment:
@baptistemesta are u ok with putting a glossary section?
Is the content ok for you?